remoterover.sh

```diff
@@ -175,6 +175,13 @@ if [[ "x${RUN_I2C}" == "x1" ]]; then
   ) || die "Adding i2c-dev to default modules failed"
 fi
 
+if [[ "x${RUN_RELAY}" == "x1" ]]; then
+  info "Adding relay script to home directory"
+  ( set -x
+  cp relay.sh root/home/$USERNAME
```

Comment:
Let's create a directory for this kind of scripts i.e.: /usr/local/bin/
Please create simlink with command to that relay.sh like:
sudo cp relay.sh /usr/local/bin/set_relay

Reply:
I think symlink is still not created. I think I also mistakenly give wrong command for that. The right one is:
sudo ln -s /usr/local/bin/remote.sh /usr/local/bin/set_relay

WIth that you can use the script as a normal command in the system.